pxd/libzfs.pxd

```diff
@@ -151,6 +151,32 @@ cdef extern from "libzfs.h" nogil:
         EZFS_DIFF
         EZFS_DIFFDATA
         EZFS_POOLREADONLY
+        EZFS_SCRUB_PAUSED
+        EZFS_SCRUB_PAUSED_TO_CANCEL
+        EZFS_ACTIVE_POOL
+        EZFS_CRYPTOFAILED
+        EZFS_NO_PENDING
+        EZFS_CHECKPOINT_EXISTS
+        EZFS_DISCARDING_CHECKPOINT
+        EZFS_NO_CHECKPOINT
+        EZFS_DEVRM_IN_PROGRESS
+        EZFS_VDEV_TOO_BIG
+        EZFS_IOC_NOTSUPPORTED
+        EZFS_TOOMANY
+        EZFS_INITIALIZING
+        EZFS_NO_INITIALIZE
+        EZFS_WRONG_PARENT
+        EZFS_TRIMMING
+        EZFS_NO_TRIM
+        EZFS_TRIM_NOTSUP
+        EZFS_NO_RESILVER_DEFER
+        EZFS_EXPORT_IN_PROGRESS
+        EZFS_REBUILDING
+        EZFS_VDEV_NOTSUP
+        EZFS_NOT_USER_NAMESPACE
+        EZFS_CKSUM
+        EZFS_RESUME_EXISTS
+        EZFS_SHAREFAILED
```

Comment:
Should we add `EZFS_RAIDZ_EXPAND_IN_PROGRESS` as well?

Reply:
Yes, we should. I have updated. Thanks.